dm/tests/dmctl_basic/conf/dm-task7.yaml

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
Why does this test case need such a complicated task configuration? Is it solely used to emulate the situation without source metadata😵?

Reply:
i just copy it from other task config like `dm-task1` to `dm-task6`, but it can check not only without source meta and also make sure other configuration item to be normal.